code/obj/item/clothing/suits.dm

```diff
@@ -1058,6 +1058,14 @@
 		icon_state = "spacecap-red"
 		item_state = "spacecap-red"
 
+/obj/item/clothing/suit/space/paramedic
+	name = "paramedic space suit"
+	icon_state = "paraspace"
+	item_state = "paraspace"
+
+	setupProperties()
+		..()
+		setProperty("space_movespeed", 0.3)
```

Comment:
Because this isn't something you can easily get round start. Its a paramedic suit, it should be relatively speedy so you can get to people quickly to heal them. As per balance concerns, it costs far more then a normal spacesuit from a mining fab (3 Fab-1, 3 Met-1, 2 Cry -1 is a normal spacesuit, this is just cloth, wire, and steel.) Matwise its similar to indy armor.

Reply:
Wall of text incoming!

Right, so I think the issue is more that this is objectively better than most spacesuits - as such, you can expect to see non-medics grabbing them if the material is available.

I've no strong objections to medbay having a "flavor" of spacesuit, I just don't think they're the one that should have an expensive one that's better than most. By making it on par with the engineering/EVA spacesuits people won't just wander around in them, as the slowdown is not insignificant, making them a pro/con choice of whether to pick one up early and have it take up an inventory slot or make as-needed.

I do have a slight objection to spacesuits being so readily available, as it somewhat trivializes breaches, but given how common those are I'm fine with the QoL improvement.

Actionable things for my approval (opinion of others may differ):

Option 1) Keep move speed as you have it, but reduce the armor to be barely present (probably 1 melee, 0 ranged) - it's a lightweight suit, it hasn't got the "chunk" of an engineering one. Put the price at similar to engineering space suits, and probably use a non-roundstart material for part of it - `CON-2` is, I believe, not covered by roundstart and is readily obtainable in large amounts by mining via Claretine, but please verify). It fills a different niche to the paramedic suit (which is less spaceworthy but more sturdy in general - better armor, better heat resistance, no slowdown at all, and can be further reinforced with an armor vest).

Option 2) Remove the suit's overrides for properties, making it the same as the default suit. Alter cost to the same as an engineering suit. Now it's a reskinned spacesuit that has built-in ProDocs rather than a built-in flashlight, which seems decently well balanced. Steps on the paramedic suit's toes still, but is *probably* viewed as inferior to the paramedic suit due to the slowdown, despite spaceworthiness.

Option 3) Why are the paramedic suit and the spacesuit different? Roll them into one. This is probably the most "drastic" change as it alters the behavior of the paramedic suit. I personally have no objections to either variant mentioned in Option 1 or Option 2 *replacing* paramedic suits. Given that move speed is typically valued higher than a point or two of armor, Option 1 is *closer* to existing behavior.

Option 3 above is definitely a balance change, and is essentially "give paramedic suits a helmet to go with them, make them manufacturable, and tweak their stats". I think that's easily the most controversial change and would need a bigger discussion, but kind of makes sense I guess?

There's also the potential for making them non-manufacturable, but putting a few of them in/around medbay at map-appropriate locations.